Add issue templates (#6)
corshatech/cast: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report
about: Create a report to help us improve
title: "[BUG]"
labels: ''
assignees: ''

---

**Describe the bug**

> A clear and concise description of what the bug is.

Reviewer: ```suggestion
A clear description of the bug.
A good bug report description answers the following three questions in only a few sentences:

1. What was I doing?
2. What did I expect to happen?
3. What actually happened?
```

**To Reproduce**

> Steps to reproduce the behavior:

Reviewer: ```suggestion
Steps to reproduce the behavior, example:
```

1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

> **Desktop (please complete the following information):**
>  - OS: [e.g. iOS]
>  - Kubeshark Version [e.g. 37.0]
>  - Helm Version [e.g. 37.0]
>  - Kubernetes Version [e.g. 37.0]

Reviewer: ```suggestion
**Environment (please complete the following information):**
 - OS of your machine: [e.g. Mac, Ubuntu]
 - CAST version: [e.g. 37.0]
 - Cloud provider: [e.g. GCP, EKS, Azure]
 - Kubeshark Version: [e.g. 37.0]
 - Helm Version: [e.g. 37.0]
 - Kubernetes Version: [e.g. 37.0]
```
